packages/dashboard-backend/src/routes/api/editors.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018-2024 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { FastifyInstance } from 'fastify';
+import { FastifyRequest } from 'fastify/types/request';
+
+import { baseApiPath } from '@/constants/config';
+import { getDevWorkspaceClient } from '@/routes/api/helpers/getDevWorkspaceClient';
+import { getToken } from '@/routes/api/helpers/getToken';
+import { getSchema } from '@/services/helpers';
+
+const tags = ['Editor Definitions'];
+
+export function registerEditorsRoutes(instance: FastifyInstance) {
+  instance.register(async server => {
+    server.get(
+      `${baseApiPath}/editors`,
+      getSchema({ tags }),
+      async function (request: FastifyRequest) {
+        const token = getToken(request);
```

Comment:
I'm not able to access the dashboard as a regular user if the user doesn't have  get permissions to list configmaps in `eclipse-che` namespace:

![image](https://github.com/eclipse-che/che-dashboard/assets/83611742/dadcbee5-4214-437c-be59-a4f98dc861d3)

Should we use dashboard service account token instead @olexii4 @akurinnoy ?

Reply:
@dkwon17 should be fixed now

Reply:
Thank you @svor , it is working now:
![image](https://github.com/eclipse-che/che-dashboard/assets/83611742/5a23538c-787e-4341-8031-cfe7c0205ee8)

Reply:
![image](https://github.com/eclipse-che/che-dashboard/assets/83611742/d9b5712b-0e04-4bd2-b7cd-16baa17b37e3)